manifests/charts/ztunnel/templates/daemonset.yaml

```diff
@@ -97,6 +97,8 @@ spec:
               fieldPath: spec.serviceAccountName
         {{- if .Values.meshConfig.defaultConfig.proxyMetadata }}
         {{- range $key, $value := .Values.meshConfig.defaultConfig.proxyMetadata}}
+        - name: ISTIO_META_CLUSTER_ID
```

Comment:
I think this can be combined with:
https://github.com/istio/istio/blob/f4b984a6466a0e9c4f98143698d3b46dda8905c5/manifests/charts/ztunnel/templates/daemonset.yaml#L76-L77

I'll check it

Reply:
Should work after https://github.com/istio/ztunnel/pull/563 get merged

Reply:
is CLUSTER_ID really the only `ISTIO_META_` env var getting appended to the proxy metadata?

Reply:
Probably so, I don't think we have as much need for this as we did in Envoy. Since we have full control over ztunnel.

Reply:
Right now, I haven't seen other `ISTIO_META_` environment variables that need to be appended. The old cluster ID works well. However, for features like `proxy-status` in istioctl, no cluster information is displayed because there is no cluster information in the metadata. I intended to let istioctl to recognize its cluster.

For example:
```
istio-1.18.0 ./bin/istioctl ps
NAME                                                    CLUSTER        CDS         LDS         EDS          RDS          ECDS         ISTIOD                      VERSION
ztunnel-5znbv.istio-system                                             IGNORED     IGNORED     IGNORED      IGNORED      IGNORED      istiod-5698b755bc-46xlm     master-2023-05-24T19-01-51
ztunnel-87jdf.istio-system                                             IGNORED     IGNORED     IGNORED      IGNORED      IGNORED      istiod-5698b755bc-46xlm     master-2023-05-24T19-01-51
ztunnel-jlbjs.istio-system                                             IGNORED     IGNORED     IGNORED      IGNORED      IGNORED      istiod-5698b755bc-46xlm     master-2023-05-24T19-01-51
details-v1-59fbfbb9c9-pqtbz.bookinfo                    Kubernetes     SYNCED      SYNCED      SYNCED       SYNCED       NOT SENT     istiod-5698b755bc-46xlm     1.19-dev
details-v1-59fbfbb9c9-xgvv4.default                     Kubernetes     SYNCED      SYNCED      SYNCED       SYNCED       NOT SENT     istiod-5698b755bc-46xlm     master-2023-05-24T19-01-51
gateway-istio-66588d4964-rrmdb.istio-ingress            Kubernetes     SYNCED      SYNCED      SYNCED       SYNCED       NOT SENT     istiod-5698b755bc-46xlm     master-2023-05-24T19-01-51
```

Reply:
Or the simpler way is to read the CLUSTER_ID env var in ztunnel, and set the proxy metadata, WDYT?